benchmark/src/main/scala/keycloak/scenario/KeycloakScenarioBuilder.scala

```diff
@@ -357,6 +357,44 @@ class KeycloakScenarioBuilder {
     this
   }
 
+  def createUser(): KeycloakScenarioBuilder = {
+    chainBuilder = chainBuilder
+      .feed(Iterator.continually(Map("username" -> randomUUID())))
+      .exec(http("Create user")
+        .post(ADMIN_ENDPOINT + "/users")
+        .header("Authorization", "Bearer ${token}")
+        .header("Content-Type", "application/json")
+        .body(StringBody("""{"username":"joinGroup-${username}"}"""))
```

Comment:
The user will always have the prefix "joinGroup-", and looking the method name this surprises me. To be able to reuse it in different scenarios, I'd like to have it without the prefix "joinGroup-". It could be a more generic one like "user-", or just no prefix.